docs/Guide.Jest.md

```diff
@@ -3,46 +3,88 @@ id: Guide.Jest
 title: Jest
 ---
 
-## Usage
+## DISCLAIMERS
 
-### 0. Use the [Getting Started](Introduction.GettingStarted.md) Guide to set up Detox
+- This guide describing installing Detox with Jest on a fresh project. If you're migrating an existing project, please apply some common sense in the process.
+- We **strongly** recommend using Jest `24.x.x`.
```

Comment:
Let's write that we officially support only Jest 24.x

Reply:
Probably, we can even tell that one more time even in README, as a version we conduct testing on, currently.